### PR TITLE
data sync: refactor -R help text

### PR DESCRIPTION
**Pull request by pared**

### CODE REVIEW

File: static/docs/commands-reference/fetch.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ for more information on how to configure the remote storage.
       -d, --with-deps       Fetch cache for all dependencies of the specified
                             target.
       -R RECURSIVE, --recursive RECURSIVE
-                            Fetch cache for subdirectories of specified directory.
+                            Fetch cache for subdirectories of current directory.
```
Comment (shcheklein) [inline static/docs/commands-reference/fetch.md]: It probably should be `the current` everywhere

**Comment by pared** [comment]
Ok, this change was introduced due to my misunderstanding of how -R flag should work, I would like to delete it. Any objections?

**Comment by efiop** [comment]
@pared No need to delete and submit a new one, you could just push new changes to this PR instead. :slightly_smiling_face:

**Comment by pared** [comment]
What I mean is that this change, in my opinion is unnecessary. Shouldn't doc stay as it is?

**Comment by efiop** [comment]
@pared Ah, right. Forgot that you've sent `-R` doc changes previously. Sure, closing.

**Pull request closed without merging**